generator/toyota/_comma.dbc

```diff
@@ -5,10 +5,10 @@ BO_ 359 STEERING_IPAS_COMMA: 8 IPAS
  SG_ SET_ME_X00 : 31|8@0+ (1,0) [0|255] "" XXX
  SG_ DIRECTION_CMD : 38|2@0+ (1,0) [0|3] "" XXX
  SG_ SET_ME_X40 : 47|8@0+ (1,0) [0|255] "" XXX
- SG_ SET_ME_X00 : 55|8@0+ (1,0) [0|255] "" XXX
+ SG_ SET_ME_X00_1 : 55|8@0+ (1,0) [0|255] "" XXX
```

Comment:
why did this change? if you want to change this, do it in another PR

Reply:
This also belongs to syntax fixes. Signal names can't be the same inside one message.

Reply:
It's a duplicate of the signal above. This message is not used anywhere in code. Still better to do it in a separate PR if we ever need to look for the change.

Reply:
Reverted back, will do in a separate PR